LICENSE.md

```diff
@@ -1 +1,295 @@
+- **Surveying and Mapping Authority of the Republic of Slovenia**: source [data](http://egp.gu.gov.si/egp/) and [license](https://creativecommons.org/licenses/by/2.5/si/legalcode)
+- **The List - Bay Area Concert Guide**: source [data](http://www.calweb.net/~skoepke/) and [license](http://www.calweb.net/~skoepke/)
+- **Turku City Government**: source [data](http://opendata.lounaistieto.fi/aineistoja/Turku_pienalueet.zip) and [license](http://www.lounaistieto.fi/blog/2015/08/18/turun-palvelualuejakotilastoalueet/)
+- **Tampere City Survey GIS**: source [data](http://opendata.navici.com/tampere/opendata/ows?service=WFS&version=2.0.0&request=GetFeature&typeName=opendata:KH_TILASTO&outputFormat=json) and [license](http://www.tampere.fi/tampereen-kaupunki/tietoa-tampereesta/avoin-data/avoin-data-lisenssi.html)
+- **Toronto Social Development, Finance & Administration Department**: source [data](http://www1.toronto.ca/wps/portal/contentonly?vgnextoid=04b489fe9c18b210VgnVCM1000003dd60f89RCRD&vgnextchannel=75d6e03bb8d1e310VgnVCM10000071d60f89RCRD) and [license](http://www1.toronto.ca/wps/portal/contentonly?vgnextoid=4a37e03bb8d1e310VgnVCM10000071d60f89RCRD)
+- **Transitland**: source [data](https://transit.land/) and [license](https://transit.land/an-open-project/contributor-agreement.html)
+- **UN/LOCODE (United Nations Code for Trade and Transport Locations)**: source [data](http://www.unece.org/cefact/locode/welcome.html) and [license](http://www.unece.org/cefact/locode/locode_since1981.html)
+- **U.S. Department of State - Humanitarian Information Unit**: source [data](https://data.humdata.org/dataset/global-lsib-polygons-detailed) and [license](https://data.humdata.org/about/license)
+- **United States Census Bureau**: source [data](https://www.census.gov/) and [license](https://www.census.gov/data/developers/about/terms-of-service.html)
+- **US ZIP Code Tabulation Area**: source [data](http://www.census.gov/geo/reference/zctas.html) and [license](https://www.census.gov/data/developers/about/terms-of-service.html)
+- **Vancouver Planning and Development Services**: source [data](http://data.vancouver.ca/datacatalogue/localAreaBoundary.htm) and [license](http://vancouver.ca/your-government/open-data-catalogue.aspx#tab19099)
+- **Washington Post**: source [data](https://web.archive.org/web/20160209065412/http://opendatadc.org/dataset/neighborhood-boundaries-217-neighborhoods-washpost-justgrimes) and [license](http://opendatadc.org/dataset/neighborhood-boundaries-217-neighborhoods-washpost-justgrimes)
+- **Yahoo! GeoPlanet (formerly Where On Earth)**: source [data](http://developer.yahoo.com/geo/geoplanet/) and [license](http://developer.yahoo.com/geo/geoplanet/data/)
+- **Who's On First**: source [data](https://whosonfirst.org/) and [license](https://github.com/whosonfirst-data/whosonfirst-data/blob/master/LICENSE.md)
```

Comment:
Same as metazen / mapzen.

Reply:
This will be tracked in a whosonfirst-data issue that I'll link to this PR once closed.

Reply:
https://github.com/whosonfirst-data/whosonfirst-data/pull/1797